proto/bluzelle.proto

```diff
@@ -1,5 +1,12 @@
 syntax = "proto3";
 
+message swarm_error
```

Comment:
Does this need to stay insync with swarmdb at all?

Reply:
Yes it should be kept the same in both projects. We already have that issue. Ideally we would have stuff like this in an external project included in both projects.

Reply:
Having said that, if the swarmdb we're talking to hasn't been updated to send these messages yet, it won't be a problem.